lidur 2 klár

Thordisalda opened the pull request:
báðar skipanaskrárnar fyrir lið 2 komnar, check it out ;)

Thordisalda committed:
lidur 2 klár

báðar skipanaskrárnar fyrir lið 2 komnar, check it out ;)

IngvarAtli commented:
Mjög flott, kóði virkar einnig hjá mer og keyrist. Ég sé reyndar ekki niðurstöðurnar þínar en niðurstöðurnar sem ég fékk makea sense svo held það megi mergea þegar Logi er búinn að kíkja yfir.

Thordisalda committed:
Textaútskýring liður 2

útskýring á hvað við erum að gera í kóðanum og hverjar niðurstöðurnar eiga vera við keyrslu.

IngvarAtli reviewed (approved):
approve, comments í conversation.

Jon-slow97 reviewed (approved):
Skipanaskrárnar virka báðar og útskýring er skýr og flott.
